features/git-hack/main_branch_rebase_tracking_branch_conflict/with_open_changes.feature

```diff
@@ -13,7 +13,14 @@ Feature: git hack: resolving conflicts between main branch and its tracking bran
       |        | local    | conflicting local commit  | conflicting_file | local content  |
     And I am on the "existing_feature" branch
     And I have an uncommitted file with name: "uncommitted" and content: "stuff"
-    When I run `git hack new_feature` while allowing errors
+    When I run `git hack new_feature`
+    Then it errors and the output ends with
+      """
```

Comment:
If you're already checking that the output ends with this string, why not just trim the newlines too and compress the height of these string blocks?

I think you might be fine with trimming the leading newline. My additional argument for trimming the trailing newlines is that I don't think you ensure much by comparing them explicitly. 

```
# A
"""
hello
there

"""

# B
"""
hello
there
"""
```
I feel that these are identical in the context of git town outputs. The trailing newlines are purely stylistic. Furthermore, if we chose to either add or remove additional trailing lines, the tests wouldn't need to be modified. Just my 2c